frontend/app/src/router.tsx

```diff
@@ -209,6 +209,17 @@ const routes: RouteObject[] = [
                     },
                   ),
               },
+              {
+                path: '/tenant-settings/webhooks',
```

Comment:
nit: call the page webhook-workers since at some point we'll likely add webhooks for workflow run events?

Reply:
hmm yeah, though I'm not sure if we want to expose the name Webhook Workers to the users as I was thinking it might be confused with regular Workers. Maybe "Webhook Apps" although that also sounds weird (Inngest uses the term app)

Reply:
we can change it later 🫠

Reply:
aka never 😂